README.adoc

```diff
@@ -58,6 +58,7 @@ Smallrye JWT supports many properties which can be used to customize the token p
 |smallrye.jwt.jwks.refresh-interval|60|JWK cache refresh interval in minutes. It will be ignored unless the `mp.jwt.verify.publickey.location` points to the HTTPS URL based JWK set and no HTTP `Cache-Control` response header with a positive `max-age` parameter value is returned from a JWK HTTPS endpoint.
 |smallrye.jwt.expiration.grace|60|Expiration grace in seconds. By default an expired token will still be accepted if the current time is no more than 1 min after the token expiry time.
 |smallrye.jwt.verify.aud|none|Comma separated list of the audiences that a token `aud` claim may contain.
+|smallrye.jwt.required.claims|none|Comma separated list of required claims for the token to be valid.
```

Comment:
@radcortez May be worth syncing it here with the `aud` text ?
```
Comma separated list of the required claims that a token must contain.
```
It takes a few small iterations to settle on the text :-), I'd not likely propose this last update if the `aud` text were not just above it :-)

Reply:
In fact even the `required` word can be dropped:
```
Comma separated list of the claims that a token must contain.
```
`must contain` implies required